mw-webapp/src/component/waysBlock/WaysBlock.tsx

```diff
@@ -8,10 +8,8 @@ export const WaysBlock = () => {
   const [ways, setWays] = useState<Way[]>([]);
 
   useEffect(() => {
-    const detachListener = WayService.onValueFromRealTimeDb(setWays);
-
     return () => {
-      detachListener();
+      WayService.getWays(setWays);
```

Comment:
the same as getDayReports

Reply:
change function. Now use return instead of callback